numba_dpex/core/codegen.py

```diff
@@ -31,8 +31,8 @@ def _optimize_final_module(self):
         # Run some lightweight optimization to simplify the module.
         pmb = ll.PassManagerBuilder()
 
-        # Make optimization level depending on config.OPT variable
-        pmb.opt_level = config.OPT
+        # Make optimization level depending on config.DPEX_OPT variable
+        pmb.opt_level = config.DPEX_OPT
 
         pmb.disable_unit_at_a_time = False
         pmb.inlining_threshold = 2
```

Comment:
TY for investigating those issues. Also while investigating for https://github.com/IntelPython/numba-dpex/issues/1152 we've also found that `        pmb.inlining_threshold = 2` causes miscompilations. Could it be disabled by default ?

Reply:
FYI, opt3 works fine one devcloud with threshold being removed and fails on opt2 with theshold. However I have an environment that fails with opt3 without threshold.